What's New Feb 2025 blog post (#1416)
withastro/astro.build: src/content/blog/whats-new-februrary-2025.mdx

- [Community updates](#community)
- [Astro in the news, migration stories, how-to articles, videos, and more!](#content)
- [Tips & Tools](#tips--tools)
- [Themes & Templates](#themes--templates)
- [Showcase](showcase)
- [Starlight in the wild](#starlight-in-the-wild)

Let's dive right into it!

## Updates from Mission Control 

- 🅱 [Stable Live Previews for Storyblok’s Astro SDK](https://www.storyblok.com/mp/announcing-stable-live-preview-for-storybloks-astro-sdk)
- 🇨🇦 [Case study: How Vocation Events brought tech company Astro together](https://blog.mtl.org/en/case-study-vocation-events)
- 🍓 [Astro Integrations with Florian Lefebvre at TheJam.dev](https://cfe.dev/sessions/jamdev2025-astro-integrations/)

## Latest Releases

Minor releases introduce new features and improvements. Check out what's new!

- [Astro 5.3](https://github.com/withastro/astro/blob/refs/heads/main/packages/astro/CHANGELOG.md#530) - faster page rendering, easier setup for experimental sessions, and more!
- [Astro 5.4](https://github.com/withastro/astro/blob/refs/heads/main/packages/astro/CHANGELOG.md#540) - remote image optimization in Markdown, enhanced security for dev and preview servers, and more!

> - [Starlight 0.32](https://github.com/withastro/starlight/blob/main/packages/starlight/CHANGELOG.md#0310) - New route data middleware, i18n API access for plugins, and more!

Reviewer: ```suggestion
- [Starlight 0.32](https://github.com/withastro/starlight/blob/main/packages/starlight/CHANGELOG.md#0320) - New route data middleware, i18n API access for plugins, and more!
```

Fix anchor link

 
## Astro Adopters

> From Porche to IKEA to Jamie Oliver, some of the biggest names you know are using Astro!

Reviewer: Should this mention instead "From Michelin to NordVPN" or something like that?

Author: I was thinking that this would be standard wording that wouldn't change, just we'd update whenever we spot a new big site.

Does that feel confusing?

Reviewer: Thought about that but was not sure. I wouldn't say confusing, but enough to make me think about it. Could be worth checking what other people think considering I'm not a native speaker?

Author: I see your point! Easy enough to change. It's a brand new section anyway. There are no rules!!

> From Porche to IKEA to Jamie Oliver, some of the biggest names you know are using Astro!

Author: ```suggestion
From Michelin to NordVPN to Scout Motors, some of the biggest names you know are using Astro!
```

export const featuredSites = [
  'www.michelin.com',
  'nordvpn.com',
  'www.scoutmotors.com',
]; 

<ShowcaseGridById ids={featuredSites} />

## Weird Wide Web

Some sites from the last month that show off what you can do with Astro that are quirky, innovative, inspiring or just plain fun!

export const monthlySites = [
  'codetv.dev',
  'isastro.pages.dev',
  'wybm-valentine-2025.netlify.app',
  'photo-splitter.vercel.app',
  'timer.bryanhogan.com',
  'www.loremipsums.org',
]; 

<ShowcaseGridById ids={monthlySites} />

## Community

Astro wouldn’t be what it is without its community. From Discord support, to documentation improvements, to translations, to code contributions, Astro welcomes it all.

### Astro Team News

> We are pleased to welcome our newest Astro core maintainer [@Fryuni](https://astro.badg.es/contributor/fryuni/)! 💛

Reviewer: ```suggestion
We are pleased to welcome our newest Astro core maintainer [@Fryuni](https://astro.badg.es/contributor/Fryuni/)! 💛
```

> [![@Fryuni Astro contributions](https://astro.badg.es/v2/contributor/Fryuni.svg)](https://astro.badg.es/contributor/ArmandPhilippot/)

Reviewer: ```suggestion
[![@Fryuni Astro contributions](https://astro.badg.es/v2/contributor/Fryuni.svg)](https://astro.badg.es/contributor/Fryuni/)
```
